src/asn1-parser.ts

```diff
@@ -9,6 +9,23 @@ function toArrayBuffer(buffer: Buffer) {
   return ab;
 }
 
+function isUnsignedInteger(buffer: Buffer) {
+  return buffer[0] === 0;
+}
+
+function pad(params: {
+  buffer: Buffer;
+  length: number;
+  element: number;
+}): Buffer {
+  const { buffer, length, element } = params;
+
+  const padding = Buffer.from(
+    [...Array(length - buffer.length)].map((x) => element)
```

Comment:
```suggestion
  const padding = Buffer.alloc(length - buffer.length, 0);
```

Could you use  `Buffer.alloc`?

Reply:
I hadn't know it, thanks! 🙏🏻